assets/settings/default.json

```diff
@@ -12,8 +12,8 @@
   "base_keymap": "VSCode",
   // Features that can be globally enabled or disabled
   "features": {
-    // Show Copilot icon in status bar
-    "copilot": true
+    // Which inline completion provider to use.
+    "inline_completion_provider": "copilot"
```

Comment:
Do we need to have some kind of deprecation / handling for this change in settings?